<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTViewer</class>
 <widget class="QMainWindow" name="CTViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CT查看器</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="DisplayWindow_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="DisplayWindow_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeCorona">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>580</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeSagittal">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>660</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeTransversal">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Transversal">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>横切面</string>
    </property>
   </widget>
   <widget class="QLabel" name="Corona">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>冠状面</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sagittal">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>660</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>矢状面</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="DisplayWindow_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
